### PR TITLE
[gts] use supports expression

### PR DESCRIPTION
**Pull request by autoantwort**

### CODE REVIEW

File: ports/gts/vcpkg.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "name": "gts",
   "version": "0.7.6",
-  "port-version": 5,
+  "port-version": 6,
   "description": "3D surfaces meshed with interconnected triangles",
   "homepage": "http://gts.sourceforge.net/",
+  "supports": "!osx",
```
Comment (LilyWangLL) [inline ports/gts/vcpkg.json:7]: Could you provide the related link or reason of doesn't support osx?

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

File: ports/gts/vcpkg.json (hunk 1 above)
Comment (autoantwort) [reply ports/gts/vcpkg.json:7]: It states that it only supports unix and you get errors like 
```
[17/37] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DGTS_BINARY_AGE=1 -DGTS_COMPILATION -DGTS_INTERFACE_AGE=1 -DGTS_MAJOR_VERSION=0 -DGTS_MICRO_VERSION=6 -DGTS_MINOR_VERSION=7 -DGTS_VERSION=.. -DHAVE_FPU_CONTROL_H -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/gts/src/645123b72d-4f56670711.clean/src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/gts/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/debug/lib/pkgconfig/../../../include/glib-2.0 -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/debug/lib/pkgconfig/../../../include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/debug/lib/pkgconfig/../../lib/glib-2.0/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.3.sdk -DPCRE_STATIC -MD -MT CMakeFiles/gts.dir/src/predicates.c.o -MF CMakeFiles/gts.dir/src/predicates.c.o.d -o CMakeFiles/gts.dir/src/predicates.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/gts/src/645123b72d-4f56670711.clean/src/predicates.c
FAILED: CMakeFiles/gts.dir/src/predicates.c.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DGTS_BINARY_AGE=1 -DGTS_COMPILATION -DGTS_INTERFACE_AGE=1 -DGTS_MAJOR_VERSION=0 -DGTS_MICRO_VERSION=6 -DGTS_MINOR_VERSION=7 -DGTS_VERSION=.. -DHAVE_FPU_CONTROL_H -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/gts/src/645123b72d-4f56670711.clean/src -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/gts/arm64-osx-dbg -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/debug/lib/pkgconfig/../../../include/glib-2.0 -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/debug/lib/pkgconfig/../../../include -isystem /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/debug/lib/pkgconfig/../../lib/glib-2.0/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.3.sdk -DPCRE_STATIC -MD -MT CMakeFiles/gts.dir/src/predicates.c.o -MF CMakeFiles/gts.dir/src/predicates.c.o.d -o CMakeFiles/gts.dir/src/predicates.c.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/gts/src/645123b72d-4f56670711.clean/src/predicates.c
In file included from /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/gts/src/645123b72d-4f56670711.clean/src/predicates.c:129:
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/gts/src/645123b72d-4f56670711.clean/src/rounding.h:23:12: fatal error: 'fpu_control.h' file not found
#  include <fpu_control.h>
           ^~~~~~~~~~~~~~~
1 error generated.
```
So it assumes that there are headers that don't exists on macOS

Comment (LilyWangLL) [reply ports/gts/vcpkg.json:7]: Thanks~

Comment (LilyWangLL) [reply ports/gts/vcpkg.json:7]: This header exist glibc platform.
http://www.gnu.org/software/gnulib/manual/html_node/fpu_005fcontrol_002eh.html

**Review by LilyWangLL — APPROVED** [review]

**Pull request merged**